utils/build-script-impl

```diff
@@ -2012,28 +2116,18 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARG
 done
 # END OF BUILD PHASE
 
+
 # Trap function to print the current test configuration when tests fail.
 # This is a function so the text is not unnecessarily displayed when running -x.
 tests_busted ()
 {
     echo "*** Failed while running tests for $1 $2"
 }
 
-for deployment_target in "${STDLIB_DEPLOYMENT_TARGETS[@]}"; do
-    case ${deployment_target} in
-        linux-* | freebsd-* | macosx-* | cygwin-*)
-            # OK, we can run tests directly.
-            ;;
-        iphoneos-* | iphonesimulator-* | appletvos-* | appletvsimulator-* | watchos-* | watchsimulator-* | android-* )
-            # FIXME: remove this
-            # echo "Don't know how to run tests for ${deployment_target}"
-            continue
-            ;;
-        *)
-            echo "Unknown compiler deployment target: ${deployment_target}"
-            exit 1
-            ;;
-    esac
+for host in "${ALL_HOSTS[@]}"; do
```

Comment:
Why did you change this to `ALL_HOSTS`?  When compiling on OS X, we are also cross-compiling for iOS simulator, for example, and we want to run tests for both OS X and iOS simulator; that is, for every stdlib target.